ui/modal/create_password_modal.go

```diff
@@ -46,10 +49,14 @@ func NewCreatePasswordModal(l *load.Load) *CreatePasswordModal {
 		passwordStrength: l.Theme.ProgressBar(0),
 		btnPositve:       l.Theme.Button(new(widget.Clickable), "Confirm"),
 		btnNegative:      l.Theme.Button(new(widget.Clickable), "Cancel"),
+		isMinimizable:    true,
 	}
 
-	cm.btnPositve.TextSize, cm.btnNegative.TextSize = values.TextSize16, values.TextSize16
-	cm.btnPositve.Font.Weight, cm.btnNegative.Font.Weight = text.Bold, text.Bold
+	cm.btnNegative.TextSize = values.TextSize16
+	cm.btnNegative.Font.Weight = text.Bold
```

Comment:
Use `text.Medium` for modal buttons according to the mockup.
```suggestion
	cm.btnNegative.Font.Weight = text.Medium
```